WebApiCoreSeed.Domain/Services/UserService.cs

```diff
@@ -26,5 +27,34 @@ public UserService(WebApiCoreSeedContext dbContext)
         {
             return await _dbContext.Users.FindAsync(userId);
         }
+
+
+        //public async Task<List<User>> GetAsync(Guid userId)

```

Comment:
Will you implement this method? If you don't need it perhaps you can remove these commented lines